node/cmd/guardiand/node.go

```diff
@@ -622,7 +625,17 @@ func runNode(cmd *cobra.Command, args []string) {
 		logger.Fatal("Please specify --nodeKey")
 	}
 	if *guardianKeyPath == "" {
-		logger.Fatal("Please specify --guardianKey")
+		// This if-statement is nested, since checking if both are empty at once will always result in the else-branch
+		// being executed if at least one is specified. For example, in the case where the singer URI is specified and
+		// the guardianKeyPath not, then the else-statement will create an empty `file://` URI.
+		if *guardianSignerUri == "" {
+			logger.Fatal("Please specify --guardianKey or --guardianSignerUri")
+		}
+	} else {
+		// If guardianKeyPath is set, set guardianSignerUri to the file signer URI, pointing to guardianKeyPath.
```

Comment:
Shouldn't you throw if both are set?

Reply:
So I decided that I would default to using `guardianKeyPath`, even if the signer URI is set. I don't have any strong feelings about this decision. Do you think it should rather throw if both are set?

Reply:
It seems like someone is confused if both are set, so I would suggest forcing them to fix their config. The release notes could / should clearly state that they don't *have* to change anything, but if they do set the new parameter, they should remove the old one.